mods/ra/rules/player.yaml

```diff
@@ -18,6 +18,7 @@ Player:
 		LowPowerSlowdown: 3
 		QueuedAudio: Building
 		SpeedUp: True
+		BuildTimeSpeedReduction: 100, 75, 60, 50
```

Comment:
Also just for clarification, technically this queue picks away every other value in the original scale. So **100**, _85_, **75**, _65_, **60**, _55_, **50** where the bold values has been kept for the WF queue.